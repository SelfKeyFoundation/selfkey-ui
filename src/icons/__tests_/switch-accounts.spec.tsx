import * as React from 'react';
import { create } from 'react-test-renderer';
import SwitchAccountsIcon from '../switch-accounts';

describe('Switch Accounts Icon', () => {
	it('should show Switch Accounts Icon', () => {
		const component = create(<SwitchAccountsIcon />);
		const tree = component.toJSON();
		expect(tree).toMatchSnapshot();
	});
});
