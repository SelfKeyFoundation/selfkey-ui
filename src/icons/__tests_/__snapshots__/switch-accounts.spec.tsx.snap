// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Switch Accounts Icon should show Switch Accounts Icon 1`] = `
<svg
  aria-hidden="true"
  className="MuiSvgIcon-root-1 Component-root-0-1-1"
  focusable="false"
  role="presentation"
  style={
    Object {
      "height": "16px",
      "width": "16px",
    }
  }
  viewBox="0 0 16 16"
>
  <g
    fillRule="evenodd"
    id="ðŸ–¥-Main-Dashboard-sai"
    stroke="none"
    strokeWidth="1"
  >
    <g
      id="300.4-Navigation-collapsed-sai"
      transform="translate(-20.000000, -1314.000000)"
    >
      <g
        id="Menu"
      >
        <g
          id="Bottom-menu"
          transform="translate(20.000000, 1263.000000)"
        >
          <path
            d="M8,51 C3.584,51 0,54.584 0,59 C0,63.416 3.584,67 8,67 C12.416,67 16,63.416 16,59 C16,54.584 12.416,51 8,51 Z M8,65.4 C4.472,65.4 1.6,62.528 1.6,59 C1.6,55.472 4.472,52.6 8,52.6 C11.528,52.6 14.4,55.472 14.4,59 C14.4,62.528 11.528,65.4 8,65.4 Z M10.4,54.6 L10.4,56.6 L7.2,56.6 L7.2,58.2 L10.4,58.2 L10.4,60.2 L13.2,57.4 L10.4,54.6 Z M5.6,57.8 L2.8,60.6 L5.6,63.4 L5.6,61.4 L8.8,61.4 L8.8,59.8 L5.6,59.8 L5.6,57.8 Z"
            id="icon-switch-accounts"
          />
        </g>
      </g>
    </g>
  </g>
</svg>
`;
